src/StoreKeeper/WooCommerce/B2C/Exports/AbstractExport.php

```diff
@@ -117,17 +121,38 @@ public function run()
                     try {
                         $this->processItem($wpObject);
                     } catch (\Throwable $exception) {
-                        $exceptions[] = $exception;
+                        $exceptions[] = $this->catchKnownExceptions($exception);
                     }
                 }
             } catch (\Throwable $exception) {
-                $exceptions[] = $exception;
+                $exceptions[] = $this->catchKnownExceptions($exception);
             }
         }
 
         return $exceptions;
     }
 
+    protected function catchKnownExceptions($throwable)
+    {
+        if ($throwable instanceof AuthException) {
+            return new ExportException(
+                esc_html__('Failed to log in. Account data or password/hash can be wrong. You also may be banned due to multiple login errors.', I18N::DOMAIN),
+                $throwable->getCode(),
+                $throwable
+            );
+        }
+
+        if (($throwable instanceof GeneralException) && 'Permission' === $throwable->getApiExceptionClass()) {
+            return new ExportException(
+                esc_html__('No sufficient rights to call the function.', I18N::DOMAIN),
```

Comment:
You should never get this error, cos probably it means you are calling wrong (not available) function, how did you arrive to it?

Reply:
Removed catching of permission exception.